public/MainEvents/Processes/Studio/StudioZip.js

```diff
@@ -0,0 +1,21 @@
+import {getProcessParams} from '../Helpers/ProcessParams.js'
+import {pack} from "../BinFiles/7zipCommands.js";
+
+
+function main(filepath, storyPath) {
+  pack(storyPath, filepath, (e) => {
+    if(e) {
+      process.stderr.write(e.toString())
+    } else {
+      process.stdout.write('success')
+    }
+  })
+}
+
+const _params_ = getProcessParams()
+
+if (_params_.length !== 2) {
+  process.stderr.write('bad args')
```

Comment:
if you create a new error message, you have to had in locale files too. 'bad-args'